Data_Transfer_Hash/readme.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: pattern
+title: data transfer hash
+folder: Data_Transfer_Hash
+permalink: /patterns/Data_Transfer_Hash/
+categories: Architectural
+tags:
+ - Performance

```

Comment:
I would suggest structural and decoupling. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags